tests/test_security.py

```diff
@@ -0,0 +1,206 @@
+from html import escape
```

Comment:
Since we removed all the `|safe` filters, I thought it would be nice to have some tests for the HTML escaping machinery

From django-cripsy-forms' [CHANGELOG](https://github.com/django-crispy-forms/django-crispy-forms/blob/main/CHANGELOG.md#major-changes-and-migration-guide):  
> Widespread use of the |safe in crispy-form templates is removed. Audit for html input in your forms especially for Field and Prepended/Appended text. To retain the previous behaviour mark text as safe in your project code using `mark_safe()`